test/arrays/ferguson/semantic-examples/valgrind-futures/4-return-alias-local.chpl

```diff
@@ -1,7 +1,7 @@
 use samples;
 proc f() {
   var A:[1..4] R;
-  var B => A;
+  ref B = A;
```

Comment:
Sorry, I pasted the wrong test name. `4-return-ref-local.chpl` is the one I was talking about.